.github/workflows/push_docker.yaml

```diff
@@ -3,7 +3,7 @@ name: Push Docker image
 on:
   push:
     branches:
-      - master
+      - multi-arch-docker-build
```

Comment:
This can be reverted back to `master` now, right?

Reply:
yes!

Reply:
Hasn't been addressed. Please do a thorough scan of the workflow file to ensure that it can land in `master` and request review.